lib/client/db/dbcmd/dbcmd.go

```diff
@@ -505,6 +510,26 @@ func (c *CLICommandBuilder) getSnowflakeCommand() *exec.Cmd {
 	return cmd
 }
 
+func (c *CLICommandBuilder) getElasticsearchCommand() *exec.Cmd {
+	args := []string{fmt.Sprintf("https://%v:%v/", c.host, c.port)}
```

Comment:
Does in case of `--tunnel`  flag the ElasticSearch client still sends the HTTPS request even if the `c.options.noTLS` option was set ?